### ISSUE TITLE
Update MITx theme to work with dogwood

### ISSUE THREAD

**Issue by pdpinch** [mitodl/salt-ops#88]
The old theme no longer works with dogwood, but the new comprehensive theming has some issues. 

I believe our plan is to move the old theme files into new locations to make them work again?

@itsbenweeks @indagation please update this description if I'm wrong.

**Comment by pdpinch** [comment]
It looks like we have two theme repos, https://github.com/mitodl/mitx-residential-theme and https://github.com/mitocw/edx-theme -- which one should we do our work in?

**Comment by indagation** [comment]
https://github.com/mitocw/edx-theme is the repo that I made some changes on. It appears that our current Stanford Theme can work with dogwood, according to documentation in github, https://github.com/edx/edx-platform/tree/named-release/dogwood.rc/themes#stanford-theming

The documentation says that you just need to add some files to the sass folder, 
lms-main.scss
lms-main-rtl.scss
lms-course.scss
lms-course-rtl.scss
lms-footer.scss
lms-footer-rtl.scss

I added those files and tried to use the theme on dogwood and it all seemed to work fine. I added the files and made a pull request. https://github.com/mitocw/edx-theme/pull/29

Ben and I decided that it would be best to just make the current stanford theme work instead of moving to the comprehensive theme right now. This was decided after several hours of attempting to make the comprehensive theme work properly. The comprehensive theme had a lot of inconsistent and quite confusing documentation, and after a lot of effort we really just figured out that it was still going to take a good amount of effort to make the comprehensive theme work properly. On top of that, it looks like in the current master they have changed how the comprehensive themes work a bit. So it would take more effort to changed things again in the future.

So it is my opinion that if the stanford theme can work with dogwood, then we should just add those six files that we need to add to make it work, and be done with it. Given the timeframe and amount of effort I think that this would be best.
